common/app/routes/settings/Settings.jsx

```diff
@@ -126,7 +126,7 @@ export class Settings extends React.Component {
       toggleMonthlyEmail,
       toggleNotificationEmail
     } = this.props;
-    if (!username && !showLoading) {
+    if (!username && showLoading) {
```

Comment:
This change is not directly needed to fix the bug, I just noticed in investigating this bug that the loader skeleton was not appearing until after the loading was completed, so in the case of a logged in user they actually never saw the loading skeleton.

Reply:
Nice catch!